src/server/resolvers/index.ts

```diff
@@ -318,6 +318,7 @@ export const resolvers: CustomResolvers<Context> = {
 				throw new UserInputError(`user ${id} (${value}) error: ${JSON.stringify(err)}`);
 
 			if (status === ApplicationStatus.Accepted) sendStatusEmail(value, ApplicationStatus.Accepted);
+			if (status === ApplicationStatus.Rejected) sendStatusEmail(value, ApplicationStatus.Rejected);
```

Comment:
Even though this should never be a problem, I think that this should be an `else if`